lib/package.gi

```diff
@@ -1192,6 +1192,9 @@ InstallGlobalFunction( ReadPackage, function( arg )
     if   Length( arg ) = 1 then
       # Guess the package name.
       pos:= Position( arg[1], '/' );
+      if pos = fail then
+        Error(Concatenation(arg[1], " is not a valid filename. Use 'LoadPackage' to load packages"));
+      fi;
```

Comment:
I think having a proper error message here is a good idea. I am not so sure this is the message we want to print. Somebody confusing ReadPackage with LoadPackage sounds like a very exotic and specific case; it seems much more likely to me that somebody mistyped a path. In that case, I'd be very confused if I was suddenty told to use `LoadPackage`.

So how about being more descriptive in the error? Something like this:
```gap
 Error(arg[1], " is not of the form <package/filepath>")'
```

What I'd really prefer was if we could deprecate the one-parameter version of `ReadPackage`. Alas, lots of packages use it.

BTW, you don't need `Concatenation` here, Error works like print and takes multiple arguments just fine.